exporter/postgres/crunchy-postgres-exporter-pg10-el6.service

```diff
@@ -36,7 +36,7 @@ start() {
 
   echo -n $"Starting postgres_exporter: "
   set -o pipefail
-  cat $QUERY_FILE_LIST | sed "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g" > $(echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\).*/\1/')
+  cat $QUERY_FILE_LIST | sed "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g" | sed "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT}/g" > $(echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\).*/\1/')
```

Comment:
you should be able to `sed -e` here too

Reply:
Fixed that to just use the sed -e option multiple times in CentOS 6 files too.